server/routers/account-recovery/fetchRecoverableAccountWallets.ts

```diff
@@ -48,6 +49,11 @@ export const fetchRecoverableAccountWallets = publicProcedure
       },
       where: {
         userId: input.userId,
+        walletPrivacySetting: WalletPrivacySetting.PUBLIC,
```

Comment:
We were listing LOST wallets too. Not a big issue though because once we merge the other PR to avoid duplicate wallets, where will be no LOST wallets, those would simply be removed. In any case, at some point in the future, this change will be needed anyway.